nixos/mixins/telegraf.nix

```diff
@@ -105,7 +105,7 @@ in
               files = [ "/var/log/telegraf/*" ];
             }
           ]
-          ++ lib.optional (lib.any (fs: fs == "ext4") config.boot.supportedFilesystems) {
```

Comment:
I spotted this after I applied a similar fix: https://github.com/nix-community/srvos/pull/390/commits/43b147e95481e25867a13e324cfb132c9fc6f5e4